project/Dependencies.scala

```diff
@@ -20,6 +20,10 @@ object Dependencies {
     "org.scalacheck" %% "scalacheck"           % "1.15.4" % Test cross CrossVersion.binary
   )
 
+  lazy val parser: Seq[ModuleID] = Seq(
+    "eu.cdevreeze.xpathparser" %% "xpathparser" % "0.8.0"
+  )
+
```

Comment:
I'd to create an object with the name of the module and inside a var like `dedicated` that aggregates all specifics deps for that module.

I'd like this approach because you wont have to touch `build.sbt` in case of deps update

```suggestion
object Dependencies {
  object XPath {
     val dedicated = Seq(parser).flatten
     private val parser: Seq[ModuleID] = Seq(
        "eu.cdevreeze.xpathparser" %% "xpathparser" % "0.8.0"
     )
  }
}
```